src/styles/sidebar/components/SelectionTabs.scss

```diff
@@ -1,70 +1,29 @@
-@use '@hypothesis/frontend-shared/styles/components/buttons';
-
-@use '../../mixins/layout';
-@use '../../mixins/utils';
 @use '../../variables' as var;
 
-.SelectionTabs-container {
-  @include layout.vertical-rhythm;
+.SelectionTabs__container {
   // FIXME: This should be a margin, and it should be handled by the parent,
   // but needs to be considered carefully because applying vertical rhythm to
   // this component's parent messes with the calculations in the virtualized
   // thread list. Needs another pass. Note also that it is `10px` (and looks
-  // unbalanced at the standard vertical rhythm size of `1em`)
+  // unbalanced at the standard vertical rhythm size of `1rem`)
   padding-bottom: 10px;
 }
 
-.SelectionTabs {
-  @include layout.row;
-  @include layout.horizontal-rhythm(20px);
-}
-
-.SelectionTabs__icon {
-  color: var.$grey-mid;
-  margin: 0 var.$layout-space--xxsmall;
-}
-
-.SelectionTabs__type {
-  @include buttons.reset;
-
-  color: var.$color-text;
-  cursor: pointer;
-  min-width: 85px;
-  min-height: 18px;
-  // Give the tab a radius to allow :focus styling to appear similar to that of buttons
-  border-radius: var.$border-radius;
+.SelectionTab {
+  margin: 0;
+  padding: 0;
+  font-weight: normal;
+  background-color: transparent;
 
-  user-select: none;
+  min-width: 5.25rem;
```

Comment:
This value was chosen because it is roughly the same as before assuming 16px font size?

Reply:
Yes, I'm being pedantic about migrating to `rem` for dimensions, `em` for text-related rules.